src/lib/modernExtend.ts

```diff
@@ -439,6 +451,11 @@ export function numeric(args: NumericArgs): ModernExtend {
         if (valueMax !== undefined) expose = expose.withValueMax(valueMax);
         if (valueStep !== undefined) expose = expose.withValueStep(valueStep);
         if (label !== undefined) expose = expose.withLabel(label);
+        if (entityCategory) {
+            if ((args.access === 'ALL' && entityCategory === 'config') || (args.access !== 'ALL' && entityCategory === 'diagnostic')) {
```

Comment:
Please move this logic to a function and also use it in `enumLookup`

Reply:
Done